packages/docs/source/base/type.md

```diff
@@ -180,7 +191,7 @@ Our type variants include two <a href="https://developer.mozilla.org/en-US/docs/
   </table>
 </figure>
 
-Our heading line-height should be used for any `font-size` set to `ms(4)` or higher.
+Our heading line-height should be used for any `font-size` set to `$size-title-2` or higher.
```

Comment:
Is there a more future-friendly rule here? Is it because they're larger than `2rem`? What determines this other than the token name?